docs/topics/loaders.rst

```diff
@@ -179,595 +114,3 @@ output processors are declared using the ``_out`` suffix. And you can also
-declaring, instantiating or using Item Loader. They are used to modify the
-behaviour of the input/output processors.
-
-For example, suppose you have a function ``parse_length`` which receives a text
-value and extracts a length from it::
-
-    def parse_length(text, loader_context):
-        unit = loader_context.get('unit', 'm')
-        # ... length parsing code goes here ...
-        return parsed_length
-
-By accepting a ``loader_context`` argument the function is explicitly telling
-the Item Loader that it's able to receive an Item Loader context, so the Item
-Loader passes the currently active context when calling it, and the processor
-function (``parse_length`` in this case) can thus use them.
-
-There are several ways to modify Item Loader context values:
-
-1. By modifying the currently active Item Loader context
-   (:attr:`~ItemLoader.context` attribute)::
-
-      loader = ItemLoader(product)
-      loader.context['unit'] = 'cm'
-
-2. On Item Loader instantiation (the keyword arguments of Item Loader
-   ``__init__`` method are stored in the Item Loader context)::
-
-      loader = ItemLoader(product, unit='cm')
-
-3. On Item Loader declaration, for those input/output processors that support
-   instantiating them with an Item Loader context. :class:`~processor.MapCompose` is one of
-   them::
-
-       class ProductLoader(ItemLoader):
-           length_out = MapCompose(parse_length, unit='cm')
-
-
-ItemLoader objects
-==================
-
-.. class:: ItemLoader([item, selector, response], \**kwargs)
-
-    Return a new Item Loader for populating the given Item. If no item is
-    given, one is instantiated automatically using the class in
-    :attr:`default_item_class`.
-
-    When instantiated with a ``selector`` or a ``response`` parameters
-    the :class:`ItemLoader` class provides convenient mechanisms for extracting
-    data from web pages using :ref:`selectors <topics-selectors>`.
-
-    :param item: The item instance to populate using subsequent calls to
-        :meth:`~ItemLoader.add_xpath`, :meth:`~ItemLoader.add_css`,
-        or :meth:`~ItemLoader.add_value`.
-    :type item: :class:`~scrapy.item.Item` object
-
-    :param selector: The selector to extract data from, when using the
-        :meth:`add_xpath` (resp. :meth:`add_css`) or :meth:`replace_xpath`
-        (resp. :meth:`replace_css`) method.
-    :type selector: :class:`~scrapy.selector.Selector` object
-
-    :param response: The response used to construct the selector using the
-        :attr:`default_selector_class`, unless the selector argument is given,
-        in which case this argument is ignored.
-    :type response: :class:`~scrapy.http.Response` object
-
-    The item, selector, response and the remaining keyword arguments are
-    assigned to the Loader context (accessible through the :attr:`context` attribute).
-
-    :class:`ItemLoader` instances have the following methods:
-
-    .. method:: get_value(value, \*processors, \**kwargs)
```

Comment:
A proposal: to avoid copy-paste, but still have docs for these methods in Scrapy itself, make them docstrings in itemloaders repo, and then use autodocs. If I'm not mistaken, this is what we've done with scrapy.Selector, when it was moved to parsel.